templates/emails/failed-preorder-authentication.php

```diff
@@ -25,7 +25,7 @@
 	</p>
 
 <?php if ( $email->get_custom_message() ) : ?>
-	<blockquote><?php echo wpautop( esc_html( wptexturize( $email->get_custom_message() ) ) ); ?></blockquote>
+	<blockquote><?php echo wp_kses( wpautop( wptexturize( $email->get_custom_message() ) ), [ 'p' ] ); ?></blockquote>
```

Comment:
Thinking about this again, I think we need to pass an empty array into the paragraph tag for this to properly work, like this:

```suggestion
	<blockquote><?php echo wp_kses( wpautop( wptexturize( $email->get_custom_message() ) ), [ 'p' => [] ] ); ?></blockquote>
```